From 8854b5c2db8703eea58daf8969cfe17dfa4c0446 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Tue, 26 Sep 2017 14:48:34 -0500
Subject: [PATCH 250/278] drivers/net: Include Dependency for Axxia NEMAC

The Axxia nemac Ethernet driver requires the Axxia MDIO
driver.  This commit makes that explicit in Kconfig.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 drivers/net/ethernet/intel/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/intel/Kconfig b/drivers/net/ethernet/intel/Kconfig
index 44d13ea..cd76a73 100644
--- a/drivers/net/ethernet/intel/Kconfig
+++ b/drivers/net/ethernet/intel/Kconfig
@@ -19,6 +19,7 @@ config AXXIA_NEMAC
 	tristate "Axxia GigE NEMAC support"
 	default y
 	depends on ARCH_AXXIA && ARM64
+	select AXXIA_MDIO
 	select MII
 	---help---
 	  This driver supports the Axxia NEMAC gigabit Ethernet controller
-- 
1.8.5.2.233.g932f7e4

